src/Clipboard.ts

```diff
@@ -53,6 +53,27 @@ export const Clipboard = {
   setString(content: string) {
     NativeClipboard.setString(content);
   },
+
+  setImage(imageUrl: string) {
+    fetch(imageUrl)
```

Comment:
Probably want to return the promise from this function:

```suggestion
    return fetch(imageUrl)
```

Reply:
not sure this one, can you explain better?

Reply:
Oh, basically this is a function that performs some asynchronous actions that take some time to complete. Right now, the function isn't returning anything. But if you return a promise (even if it's a void promise), then when using this function we can wait for the image to finish copying to the clipboard _then_ do something else.

```javascript
Clipboard.setImage(myURL).then(() => doSomethingElse());
```